### ISSUE TITLE
flashblades updated to 3.1.11 yesterday and now http://localhost:9491/metrics/flashblade/array is not working

### ISSUE THREAD

**Issue by TyfoidKId** [PureStorage-OpenConnect/pure-exporter#53]
it just gives me this 
_server returned HTTP status 500 INTERNAL SERVER ERROR_

the odd thing is these are still working 

http://localhost:9491/metrics/flashblade/clients
http://localhost:9491/metrics/flashblade/usage

not sure where to even start

**Comment by TyfoidKId** [comment]
So poking at this some more and it seems the api token for my readonly user has gone missing. And when I tried to recreate the api token for that user in the gui I got an error at the bottom of the popup window saying the user couldn't be found and when I hit the ok button to finish the process the user disappeared all together from gui. I guess the good news is now all of my api metrics for that flashblade are now failing (clients and usage) so that's why it was not working on the array metrics but how did the other two continue on if the token was broken?
